src/components/EmployeeDashboard/charts/MonthlyIssueLineChart.jsx

```diff
@@ -0,0 +1,258 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ResponsiveLine } from '@nivo/line';
+import { linearGradientDef, select } from '@nivo/core';
+
+function MonthlyIssueLineChart(props) {
+  const data = props.data || [
+    {
+      id: 'OPEN',
+      color: 'rgb(255, 128, 0)',
+      data: [
+        {
+          x: 'Jan',
+          y: 187,
+        },
+        {
+          x: 'Feb',
+          y: 230,
+        },
+        {
+          x: 'Mar',
+          y: 259,
+        },
+        {
+          x: 'Apr',
+          y: 15,
+        },
+        {
+          x: 'May',
+          y: 67,
+        },
+        {
+          x: 'June',
+          y: 170,
+        },
+        {
+          x: 'July',
+          y: 32,
+        },
+        {
+          x: 'Aug',
+          y: 170,
+        },
+        {
+          x: 'Sep',
+          y: 116,
+        },
+        {
+          x: 'Oct',
+          y: 1,
+        },
+        {
+          x: 'Nov',
+          y: 40,
+        },
+        {
+          x: 'Dec',
+          y: 300,
+        },
+      ],
+    },
+    {
+      id: 'IN PROGRESS',
+      color: 'rgb(16, 104, 227)',
+      data: [
+        {
+          x: 'Jan',
+          y: 187,
+        },
+        {
+          x: 'Feb',
+          y: 230,
+        },
+        {
+          x: 'Mar',
+          y: 259,
+        },
+        {
+          x: 'Apr',
+          y: 15,
+        },
+        {
+          x: 'May',
+          y: 67,
+        },
+        {
+          x: 'June',
+          y: 170,
+        },
+        {
+          x: 'July',
+          y: 32,
+        },
+        {
+          x: 'Aug',
+          y: 170,
+        },
+        {
+          x: 'Sep',
+          y: 116,
+        },
+        {
+          x: 'Oct',
+          y: 1,
+        },
+        {
+          x: 'Nov',
+          y: 40,
+        },
+        {
+          x: 'Dec',
+          y: 300,
+        },
+      ],
+    },
+    {
+      id: 'RESOLVED',
+      color: 'rgb(255,255,255)',
+      data: [
+        {
+          x: 'Jan',
+          y: 187,
+        },
+        {
+          x: 'Feb',
+          y: 230,
+        },
+        {
+          x: 'Mar',
+          y: 259,
+        },
+        {
+          x: 'Apr',
+          y: 15,
+        },
+        {
+          x: 'May',
+          y: 67,
+        },
+        {
+          x: 'June',
+          y: 170,
+        },
+        {
+          x: 'July',
+          y: 32,
+        },
+        {
+          x: 'Aug',
+          y: 170,
+        },
+        {
+          x: 'Sep',
+          y: 116,
+        },
+        {
+          x: 'Oct',
+          y: 1,
+        },
+        {
+          x: 'Nov',
+          y: 40,
+        },
+        {
+          x: 'Dec',
+          y: 300,
+        },
```

Comment:
(Does not need to be edited) consider putting data like this in an external file and then importing it in the future if it takes up 50+ lines.